docs/readme_link.md

```diff
@@ -0,0 +1,2 @@
+```{include} ../README.md
+```
```

Comment:
Can we fix the Black Code Style badge?

<img width="603" alt="Screenshot 2024-06-17 at 10 08 44 AM" src="https://github.com/onecodex/onecodex/assets/1847232/a55a1172-eb9a-4d00-8e33-a97660d01f92">